indigo_app/templates/indigo_app/tasks/tabbed_layout.html

```diff
@@ -6,6 +6,12 @@
   <div class="container">
     <nav class="nav nav-tabs">
       {% block main-header-tabs %}
+      <a class="nav-item nav-link {% if view.tab == 'projects' %}active{% endif %}" href="{% url 'taxonomy_project_list' %}">
```

Comment:
The root concept is the taxonomy, not projects. So prefer `taxonomy` is a keyword (and as a label where necessary).

Projects are implemented as part of the taxonomy, but the system won't know about them explicitly.

Reply:
@longhotsummer I've changed the view names with using topics as the key word